internal/clients/github/client.go

```diff
@@ -55,37 +56,60 @@ query Login {
 	return resp.Viewer.Login, nil
 }
 
-func (c *Client) ListViewerProjects() []ProjectV2 {
+func (c Client) ListViewerProjects(first option.Option[int], after option.Option[ProjectCursor]) ([]ProjectV2, error) {
 	_ = `# @genqlient
-query ViewerProjectsV2($first: Int! = 10, $after: String) {
+query ViewerProjectsV2($first: Int!, $after: String) {
   viewer {
-	projectsV2(first: $first, after: $after) {
+    projectsV2(first: $first, after: $after) {
       edges {
-		cursor
+        cursor
         node {
           id
           title
+          number
+          url
+          creator {
+            login
+            url
+          }
         }
       }
     }
   }
 }`
 
-	const (
-		first = 10
-		after = ""
-	)
-
-	_, err := graphql.ViewerProjectsV2(context.Background(), c.client, first, after)
+	//nolint:gomnd
```

Comment:
you can create parameter with integer value, it should help you resolve the issue with the linter

Reply:
Ok, ill make the `first` arg be a plain int not option.